common/src/main/java/kin/sdk/migration/common/exception/AccountNotFoundInListOfAccounts.java

```diff
@@ -0,0 +1,9 @@
+package kin.sdk.migration.common.exception;
+
+public class AccountNotFoundInListOfAccounts extends MigrationFailedException {
```

Comment:
Exceptions should use the "Exception" suffix. .eg: AccountNotFoundInListOfAccountsException.
I would also shorten the class name to AccountNotFoundException for readability.

Reply:
The problem is that we already have that exception which is coming from the sdk when the account is not found in the blockchain.
Also inn this case we are talking locally.
I will try to find a better name and put the exception suffix